lib/product/types.go

```diff
@@ -185,6 +226,7 @@ type Package struct {
 // Metadata is the top level metadata the describes the automate
 // collections and its packages.
 type Metadata struct {
-	Packages    []*Package    `json:"packages"`
-	Collections []*Collection `json:"collections"`
+	Packages        []*Package       `json:"packages"`
+	DeletedPackages []DeletedPackage `json:"deleted_packages"`
```

Comment:
I'm not seeing where these deleted packages are actually used? Mind pointing me in the right direction?

Reply:
We're not actually using them anywhere in the go code right now, but I wanted  them to be in the product.meta, and the parser for that sets `DisallowUnknownFields()`, so we have to valid stuff for it here.

Reply:
👍 Just wanted to make sure I wasn't missing something here.